src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.macios.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.Runtime.InteropServices;
+using CoreGraphics;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Drawing view service
+/// </summary>
+public static partial class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from lines
+	/// </summary>
+	/// <param name="lines">Drawing lines</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Task<Stream> GetImageStream(in IList<DrawingLine> lines, in Size imageSize, in Color backgroundColor)
```

Comment:
Looks like the only place we need to have an `async` execution is on `Windows`, so can we change this API to `ValueTask` instead of `Task`?